Makefile

```diff
@@ -14,21 +17,21 @@ endif
 all: compile test readme clean
 
 %.elc:%.el
-	$(EMACS) -batch -L . -l test/test-bootstrap.el -f batch-byte-compile $(ELS)
```

Comment:
I'm not sure why this didn't have `-Q` and only a single `-` on `--batch`, but I assumed they were an oversight.